### DISCUSSION TITLE
上課建立的database打開怎麼沒東西？

### DISCUSSION THREAD

**Discussion by enoch-lightning** [1101-GS3073-C/Python-Community-GS3073-C#293] [Q&A]
在:two: :zero: :two: :two: /:three:/:two: :two:(二)上課時，老師帶我們使用SQLite建立了一個名為linebot.db的database(資料庫)。我發現一個問題：如果我將sqlitestudio-3.3.3(1)與linebot.db放在同一層，如下所示，sqlitestudio-3.3.3(1)中的SQLiteStudio中的SQLiteStudio.exe可以看到linebot.db裡面的東西。
![image](https://user-images.githubusercontent.com/86505454/160281954-717f57be-811f-4f41-9ccc-d0c2457ddf24.png)
![image](https://user-images.githubusercontent.com/86505454/160282180-021338ec-24fc-44ae-bdb9-bf74df23758d.png)
![image](https://user-images.githubusercontent.com/86505454/160282193-d656a34e-0088-4c57-8da7-66eef8dc9ba1.png)

![image](https://user-images.githubusercontent.com/86505454/160282101-b470d4b6-9bfb-4d0f-8f3c-433b014ccb7a.png)

但如果我把linebot.db放到另外一個資料夾，如下
![image](https://user-images.githubusercontent.com/86505454/160282033-ef541b55-ced5-4dc0-8b92-98512a92709d.png)
打開下載中的sqlitestudio-3.3.3(1)中的SQLiteStudio中的SQLiteStudio.exe就看不到linebot.db的內容了，情況如下。
![image](https://user-images.githubusercontent.com/86505454/160282079-3480513b-930f-41e7-af65-6becbaeaad7f.png)
不知道問題出在哪裡。

**Comment by timchen1216** [comment]
不就單純是你更改路徑
然後它找不到檔案嗎┐(´д`)┌

**Reply by chia-kai-chang** [reply]
看錯誤訊息確實是檔案路徑找不到了， @enoch-lightning 再確認一下
